sentry-tracing/src/lib.rs

```diff
@@ -13,42 +13,65 @@
 //! in the tracing system and generate events, thus be relayed to this crate. It
 //! effectively replaces `sentry-log` when tracing is used.
 //!
-//! ## Examples
+//! # Examples
 //!
 //! ```rust
+//! use std::time::Duration;
+//!
+//! use tokio::time::sleep;
 //! use tracing_subscriber::prelude::*;
 //!
-//! tracing_subscriber::registry()
-//!     .with(tracing_subscriber::fmt::layer())
-//!     .with(sentry_tracing::layer())
-//!     .try_init()
-//!     .unwrap();
+//! #[tokio::main]
+//! async fn main() {
+//!     let _guard = sentry::init(sentry::ClientOptions {
+//!         dsn: "<dsn>".parse().ok(),
```

Comment:
Its okay to have this, but IMO the examples should rather fall back to grabbing `SENTRY_DSN` from the env, which they do when this is left as `None`/default.